ProcessMaker/Http/Controllers/Designer/ProcessController.php

```diff
@@ -32,7 +32,8 @@ public function show(Process $process = null)
             request()->session()->flash('_alert', json_encode(['danger', __('The process was not found.')]));
             return view('processes.index');
         }
+        $title = $process->name;
 
-        return view('designer.designer', compact('process'));
+        return view('designer.designer', compact(['process', 'title']));
```

Comment:
Can't we just use $process->title? Do we need a second variable?

Reply:
yes, because in layout.blade.php wait variable "$title" for title of tab.
![image](https://user-images.githubusercontent.com/1747025/44090103-04caac18-9f97-11e8-8966-0467a672d637.png)